application/forms/Config/UserBackend/LdapBackendForm.php

```diff
@@ -108,35 +114,6 @@ public function createElements(array $formData)
                     'formnovalidate'    => 'formnovalidate'
                 )
             );
-
-            if ($this->getElement('discovery_btn')->isChecked()) {
-                $connection = ResourceFactory::create(
-                    isset($formData['resource']) ? $formData['resource'] : reset($this->resources)
-                );
-
-                try {
-                    $capabilities = $connection->bind()->getCapabilities();
-                    $baseDn = $capabilities->getDefaultNamingContext();
-                    $hasAdOid = $capabilities->isActiveDirectory();
-                } catch (Exception $e) {
-                    $this->warning(sprintf(
-                        $this->translate('Failed to discover the chosen LDAP connection: %s'),
-                        $e->getMessage()
-                    ));
-                }
-            }
-        }
-
-        if ($isAd || $hasAdOid) {
```

Comment:
@Al2Klimov Removing this block breaks re-population of default configurations. Especially in case of AD backends. (Try opening a AD backend configuration in Icinga Web 2)